### PR TITLE
release 5.1.1

### PR DESCRIPTION
**Pull request by michaelpacer**
This is largely based off of #9 (excepting that now it has an appropriate version number).

### CODE REVIEW

**Comment by michaelpacer** [comment]
Hrm… I'm not sure what's wrong here (or how to solve it). 

The circle ci seems to have everything to do with the permissions and authentication issues.

I don't know why appveyor got dropped. 

But in case it was because of an out-of-sync autogenerated binstar, I wanted to delete it so that if the trigger was the absence of the binstar to generate one. I see now that that's not the case. They must somehow be being set automatically for travis.ci, but the same does not hold for appveyor. …I wonder if this relates to the registration problems that I had yesterday.

File: conda-forge.yml [hunk 1]
```diff
@@ -3,4 +3,4 @@ travis:
     BINSTAR_TOKEN: cqykPcITK8G6aXGV8PndR/hFf8t6mPy1hICU2ylhsVhpNxoKMHAMbew0xoL1iIQZ1NEJddRFXi2zlJfW30hxddF08UPOZyKRxOPP8sasWud/YzrdauaNkHZ9x3p2Q8GWrIpjfWlrXBi7Xihz9y9zhUCWkU2d87kc9x8cW7h4xOFbxSUR9dChFv1AI2/Dzb21ByhmhSFtq546WmpaxrISnX3yICWr444R4tiRUJWYULdjEL0m5t1pWbOk2B84flrnFOsirFmlhavcH3USLN+L54iixZ5Mt0W80SNGQySz3h047nh4YY1yXA/uu9D4wJEAszU8Aev7y9pUygP/rMA4hUCFz+cGtYKifJyWiwhgqFboLFS7Dda0d8obPpQkIL5liSDlBZcGhEiJ7BsH/2UXO+Q1Qmx01iMrGqqltYAnvh8w9W1d71VBUuU5bpopCLuJwy1xMJwSf6SwdOQsiy7woR/knb0vvm2zKSki1/BYzkisJ/rLY+8vu/OTdpBK+h7WLAxslGjDa5XjJNT1Iv6z2gUm1s0tLh7u2d8hqsZrVz7zmuWJ+ktKKgp7iQFW8I5T/AWkl7iXYzmvRP6y14Bn+dp8qcQjxLYzzDn36hPmiwvdoPWQ0p4pU9iSt2ipzV3EWUbBWoRCVE9WgPbFaaFCOuG3eRAVpd3ts474p7z1fOU=
 appveyor:
   secure:
-    BINSTAR_TOKEN: MP4hZYylDyUWEsrt3u3cod2sbFeRwUziH02mvQOdbjsTO/l1yIxDkP/76rSIjcGC
+    BINSTAR_TOKEN:
```
Comment (jakirkham) [inline conda-forge.yml]: Please don't delete this.

Comment (jakirkham) [inline conda-forge.yml]: Also after adding it back, please re-render.

**Comment by jakirkham** [comment]
Please disable CircleCI on your fork and then repush to get it to run on the org's CircleCI.

**Comment by jakirkham** [comment]
The issue with AppVeyor does not relate to the Binstar token. Instead AppVeyor periodically has issues with GitHub rate limiting that cause the webhook to fail. GitHub rate limiting occurs for 1hr intervals. So if this happens, it is best to wait and/or ping core. We're looking at ways to mitigate this, but it may still occur with lower frequency.

xref: https://github.com/conda-forge/conda-forge.github.io/issues/314

**Comment by jakirkham** [comment]
Also retriggered AppVeyor manually.

**Comment by michaelpacer** [comment]
Hi @jakirkham, I'm looking through the docs and I'm not finding how to disable circleci anywhere officially. 

The only thing I can infer is to comment out the `circle.yml`, which seems to be the only file that is loading `checkout_merge_commit` which also seems to be where most of the circleci "action" is happening. So – hopefully – this works.

**Comment by jakirkham** [comment]
So just go to settings (gear) on the CircleCI project. It should take you to the Overview settings. Then select either Unfollow or Stop Building. This should disable CircleCI on your fork. Please let me know when you have done this.

**Comment by michaelpacer** [comment]
OH! And here I was trying to figure out where to define it in the `.yml` files. Oops!

**Comment by jakirkham** [comment]
Honestly there is very little need to muck with any of the yml files directly. Generally there is very little need to muck with the CIs either. Nearly all changes occur in the recipe directly. `conda-smithy` then propagates those changes out to the relevant files when re-rendering.

**Comment by michaelpacer** [comment]
@jakirkham Thank you for helping with all this. It seems that some of these things are hard to know without a lot of experience (especially when something seems to be broken). 

I'll keep that heuristic in mind though going forward.

BTW, happy to help with the documentation if you point me in the right direction. :)

**Comment by jakirkham** [comment]
Not a problem. Sorry this can be a bit daunting at first. There's a lot more infrastructure in conda-forge compared to a typical open source project, which probably adds to the confusion and increases the chances of something being broken. 😄 Would certainly appreciate feedback on how to make it less daunting.

Also any help with the docs is certainly welcome. Here is the [code]( https://github.com/conda-forge/conda-forge.github.io/tree/master/docs ) for them and they are hosted on this [page]( http://conda-forge.github.io/docs/ ). @scopatz has been really championing the effort of getting things documented. Certainly feel free to add any bits of wisdom that you find relevant.

**Comment by minrk** [comment]
Looks like Circle's all set now. Thanks, @michaelpacer!

**Pull request merged**